tests/contexts/index.js

```diff
@@ -0,0 +1,35 @@
+const invalidContexts = {
+  veresOne: {
+    url: 'https://w3id.org/veres-one/v1',
+    value: require('./veresOnev1.jsonld')
+  },
+  did: {
+    url: 'https://w3id.org/did/v0.11',
+    value: require('./didv0.jsonld')
+  },
+  valid: {
+    url: 'https://w3id.org/credentials/v1',
+    value: require('./credentials-v1.jsonld')
```

Comment:
If this is a valid context, no sense in duplicating it in the `tests/` folder, you can just export it from `../lib/contexts`, right?

Reply:
good point